### PR TITLE
Added Sundarbazar Municipality's Website to websites.json

### PR DESCRIPTION
**Pull request by ankman007**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [X] Added a new website
- [ ] Other(s)

### CODE REVIEW

**Review by susanhsrestha — COMMENTED** [review]
A single PR must contain at least two websites.

**Comment by ankman007** [comment]
> A single PR must contain at least two websites.

I've added one more website. It should be good now.

File: data/websites.json [hunk 1]
```diff
@@ -41,15 +41,20 @@
       "description": "Official Website of Bedkot Municipality"
     },
     {
-      "name": "Belauri Municipality",
-      "url": "https://belaurimun.gov.np/",
-      "description": "Official Website of Belauri Municipality"
+      "name": "Besishahar Municipality",
```
Comment (susanhsrestha) [inline data/websites.json:44]: Here, you have altered earlier data.

File: data/websites.json [hunk 2]
```diff
@@ -41,15 +41,20 @@
...
+      "url": "https://besishaharmun.gov.np/",
+      "description": "Official Website of Besishahar Municipality"
     },
     {
       "name": "Belbari Municipality",
       "url": "https://belbarimun.gov.np/",
       "description": "Official Website of Belbari Municipality"
     },
+    {
+      "name": "",
```
Comment (susanhsrestha) [inline data/websites.json:54]: You have left the duplicate portion exactly as it is.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

File: data/websites.json (hunk 2 above)
Comment (ankman007) [reply data/websites.json]: Sorry, my bad I forgot to remove the unnecessary file changes that I made previously. Should be good now. Thanks.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
The PR should contain atleast 2 websites .

**Review by thenishantsapkota — APPROVED** [review]
LGTM!

**Review by susanhsrestha — APPROVED** [review]

**Pull request merged**